<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WebEngineClass</class>
 <widget class="QMainWindow" name="WebEngineClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>755</width>
    <height>457</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WebEngine</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="frame_left">
      <property name="minimumSize">
       <size>
        <width>40</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>40</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="QFrame" name="frame_top">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>40</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QPushButton" name="pushButton_enter">
         <property name="geometry">
          <rect>
           <x>460</x>
           <y>20</y>
           <width>21</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>E</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_refresh">
         <property name="geometry">
          <rect>
           <x>480</x>
           <y>20</y>
           <width>21</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>S</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_debug">
         <property name="geometry">
          <rect>
           <x>500</x>
           <y>20</y>
           <width>21</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>D</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_sendJs">
         <property name="geometry">
          <rect>
           <x>520</x>
           <y>20</y>
           <width>75</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Qt SendJs</string>
         </property>
        </widget>
        <widget class="QComboBox" name="comboBox_url">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>20</y>
           <width>431</width>
           <height>22</height>
          </rect>
         </property>
         <property name="editable">
          <bool>true</bool>
         </property>
         <item>
          <property name="text">
           <string>file:///./web/vue/index001.html</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>file:///./web/vue/version.html</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>https://html5test.com/</string>
          </property>
         </item>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="HWebView" name="webEngineView">
             <property name="url">
              <url>
               <string>about:blank</string>
              </url>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QWebEngineView" name="webEngineView_debug">
             <property name="url">
              <url>
               <string>about:blank</string>
              </url>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QWebEngineView</class>
   <extends>QWidget</extends>
   <header location="global">QtWebEngineWidgets/QWebEngineView</header>
  </customwidget>
  <customwidget>
   <class>HWebView</class>
   <extends>QWebEngineView</extends>
   <header>HWebView.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="WebEngine.qrc"/>
 </resources>
 <connections/>
</ui>
